server/grpc_service.go

```diff
@@ -1361,6 +1500,34 @@ func (s *GrpcServer) UpdateServiceGCSafePoint(ctx context.Context, request *pdpb
 	}, nil
 }
 
+// GetGCSafePointV2 implements gRPC PDServer.
+// Note: we need latest version of kvproto/master, but there was earlier commit https://github.com/pingcap/kvproto/pull/1111
```

Comment:
@ystaticy, I need the latest version of pingcap/kvproto@master, but it includes your earlier commit https://github.com/pingcap/kvproto/pull/1111 whose server side implementation hasn't been merged, so I added the following empty methods to avoid compile error.